_data/events/73.yml

```diff
@@ -0,0 +1,34 @@
+type: "meetup"
+number: 73
+date: 2025-02-06
+host: infraspeak
+agenda:
+  - type: intro
+  - type: talk
+    title: Road to Continuous Delivery
```

Comment:
The title should be "The Road to Continuous Delivery: Challenges, Wins, and Lessons Learned".

Reply:
Done.